e2e/upgrades/upgrades_test.go

```diff
@@ -100,6 +100,59 @@ func TestUpgradesMinorVersion(t *testing.T) {
 	steps.Run(t)
 }
 
+// TestUpgradesMinorVersion tests a minor version bump
+func TestUpgradesMajorVersion20to21WithCockroachDbVersion(t *testing.T) {
+
+	// We are testing a Minor Version Upgrade with
+	// partition update
+	// Going from v20.2.10 to v21.1.0
```

Comment:
Is this a minor or major version upgrade?

Reply:
fixed :)